### DISCUSSION TITLE
Discord Forms Solution

### DISCUSSION THREAD

**Discussion by felixfaisal** [EddieHubCommunity/support#1273] [Show and tell]
An In Discord Form Manager allowing users to have control over their data 
Demo Video - [Youtube Video](https://youtu.be/yiLA9oJ-O-s)
Git Repo - [Formica](https://github.com/felixfaisal/formica.git)

I made this project with a team of 3 including me over the last three weeks, It'd be great if you could review it :)

**Comment by schmelto** [comment]
Nice work @felixfaisal

Maybe you can open an issue for reviewing your project. There is an issue template for this. 

![Screenshot_20210410_162553.jpg](https://user-images.githubusercontent.com/30869493/114273189-7738c080-9a19-11eb-9930-c0e843a787b3.jpg)

**Reply by felixfaisal** [reply]
Yeah, But I also wanted to share the demo video majorly on Discord, So I opened a Discussion here. I'll do that as well. Thanks :D
